ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java

```diff
@@ -42,8 +42,8 @@ public void bookRequestAppointmentTest() throws Exception {
         ClinicianFacingPatientDashboardPage patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         RequestAppointmentPage requestAppointmentPage = patientDashboardPage.clickOnRequest();
         requestAppointmentPage.enterAppointmentType("Oncology");
-        requestAppointmentPage.enterValue("0");
-        requestAppointmentPage.selectUnits("Day(s)");
+        requestAppointmentPage.enterMinimumValue("0");
+        requestAppointmentPage.selectMinimumUnits("Day(s)");
```

Comment:
Refactored the methods within the `RequestAppointmentPage` for better java naming convention